test/libsolidity/semanticTests/literalSuffixes/suffix_making_view_external_call.sol

```diff
@@ -0,0 +1,41 @@
+function suffix(int32 x) pure suffix returns (int32) {
```

Comment:
This is a really weird test case that has zero relation to suffices...

Reply:
It's intentionally trying to do weird things, because technically they're allowed. I wanted to try it to see it it will actually work and it does not hurt to have some weirder test cases.

Reply:
The point is that this test does not test anything specific to suffices at all. It's testing something completely unrelated and then just straying in a suffix on top. I'm not sure what's the point of tests like that.

Reply:
The point is an external call to a `view` function. That call happens directly inside the suffix definition so it's relevant to the feature. You can assume that it should work, but it's still a valid black-box test. The fact that suffixes are no different from functions in that respect is just an assumption and something that should theoretically be tested. Of course, I'm not doing extensive black-box testing here because it would make the test surface unreasonable, but I don't think that including some tests of this kind is a bad idea.

But well, I'll remove it if it bothers you. IMO it's nice to have but it's not like it's essential here or anything. The codegen changes are straightforward enough that I think there's little doubt that suffixes are indeed properly implemented as functions.

Reply:
So since ``suffix`` functions can happen in any kind of function that is part of an inheritance tree, we now need a test for any inheritance tree in which a suffix function can occur :-)?
This is **not** related to the feature. If you apply a definition of "relevant to the feature" that is broad enough to include this, then you need to test all possible strings of programs for any change we do :-).

This test in particular didn't bother me that much, but I'm generally bothered by having several thousand lines of mostly unrelated tests ;-). But yeah, it's fine for now - but please try to add *actually relevant* tests only in the future :-).